requirements/edx/base.txt

```diff
@@ -35,6 +35,7 @@ django-threaded-multihost==1.4-1
 django-method-override==0.1.0
 djangorestframework==2.3.14
 django==1.4.18
+elasticsearch<1.0.0
```

Comment:
It's currently used in forums, and we don't want a separate installation for this - elasticsearch environments separate information by creating different indexes, and scale by configuring the cluster.

The cluster we currently have is using the earlier version. Almost everything works with the newest client except for one specific call, so we are pinning the client as recommended by elasticsearch to be compatible with the server version. When we upgrade the server, we will want to update the client.